### DISCUSSION TITLE
Fit model with multiple timeseries

### DISCUSSION THREAD

**Discussion by ghost** [autogluon/autogluon#3473] [General]
Hi,
Is it possible to create a model that makes predictions from a set of time series ?
Thanks.

**Comment by liyiersan** [comment]
Yes, different item_ids mean different time series. 
More details can refer to https://auto.gluon.ai/0.7.0/tutorials/timeseries/forecasting-indepth.html#which-forecasting-models-are-available-in-autogluon.
the content are as follows:
If the dataset consists of multiple time series, we fit a separate local model to each time series.
Global models are machine learning algorithms that learn a single model from the entire training set consisting of multiple time series.
